### ISSUE TITLE
mpv window border visible

### ISSUE THREAD

**Issue by Garulfo** [SegoCode/AutoWall#4]
Hello, for any reasons, when I use AutoWall, the window border of the video player is visible in the background 
![image](https://user-images.githubusercontent.com/7861555/101163286-b30ae600-3633-11eb-9846-c33261060144.png)

This is really really strange because when I perform manually every steps one by one in a cmd terminal, it works well : 
```
cd "C:\Apps\AutoWall\mpv\"
"C:\Apps\AutoWall\weebp\wp.exe" run mpv "C:\Apps\AutoWall\VideosHere\KagangtuyaBlackHole.mov" --loop=inf --player-operation-mode=pseudo-gui --force-window=yes --no-audio
cd "C:\Apps\AutoWall\weebp\"
wp add --wait --fullscreen --class mpv

```

Thank you very much for your app, I hope you can help :)

**Comment by SegoCode** [comment]
Hey,
Can you give me more information about? like.. Windows version, AutoWall version,  is reproducible only on the "Apply" button? 

You know how AutoWall works right? when run the bat file create in `%appdata%\Microsoft\Windows\Start Menu\Programs\Startup` (When check "Set on windows startup") still the same issuse?

**Comment by Garulfo** [comment]
![image](https://user-images.githubusercontent.com/7861555/101202913-39421f00-366a-11eb-8864-b5bd72c670f4.png)
I think I used the latest version of AutoWall, I downloaded from your github yesterday.
But anyway I don't think this is related to AutoWall itself because when I run the bat file generated the bug is reproduced.
But if cmd prompt, I copy paste a line after the other it works well. To be frank this is really weird ...

**Comment by SegoCode** [comment]
Mmm.... really weird yea... 

Try this,  create a bat file, and before the `wp add --wait --fullscreen --class mpv` put this `ping 127.0.0.1 -n 3 > nul`  and run the bat.

Same issuse for other format videos on "Apply" button?

What happens if you paste `https://www.youtube.com/embed/lRTtMcx6rSM?autoplay=1&loop=1&mute=1&playlist=lRTtMcx6rSM` in text field and press apply?

**Comment by Garulfo** [comment]
> ping 127.0.0.1 -n 3 > nul

it works! incredible :)

>Same issuse for other format videos on "Apply" button?

yeah ... I tried mov, gif, mp4 

>What happens if you paste https://www.youtube.com/embed/lRTtMcx6rSM?autoplay=1&loop=1&mute=1&playlist=lRTtMcx6rSM in text field and press apply?
it works as well, no borders

I hope it'll help :)

**Comment by SegoCode** [comment]
Nice! 
Really weird, I'm not sure why it happens, but mpv sometimes has a delay when it runs.
I will have to investigate or change video player.

Thx for the feedback 💕

**Comment by SegoCode** [comment]
Fixed in 1.5 using mpv.conf
